Ska/engarchive/update_client_archive.py

```diff
@@ -476,13 +476,14 @@ def update_full_h5_files(dat, logger, msid_files, msids, opt):
 def get_full_data_sets(ft, index_tbl, logger, opt):
     # Iterate over sync files that contain new data
     dats = []
-    for date_id, filetime0, filetime1, row0, row1 in index_tbl:
+    for row in index_tbl:
```

Comment:
And sure 'row' is fine.  I probably overthought it... I avoided 'row' as the var just because the lines of the index table defined row spaces.